---
version: 1
interactions:
- request:
    body: <CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><LocationConstraint>fr-par</LocationConstraint></CreateBucketConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;host;x-amz-acl;x-amz-content-sha256;x-amz-date,
        Signature=423d4125affaf54cee9ef7d76e144645874d094e60cde1703d7ce4e2c4759cd7
      Content-Length:
      - "150"
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Content-Sha256:
      - 2cb57fad7b7168921a4c94426cfcb9ee2953f126430595df844e22d50f029060
      X-Amz-Date:
      - 20201203T142438Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 03 Dec 2020 14:24:39 GMT
      Location:
      - /test-acc-scaleway-object-bucket-cors-empty-origin
      X-Amz-Id-2:
      - tx9d92ed5988fd4a3ea28e6-005fc8f526
      X-Amz-Request-Id:
      - tx9d92ed5988fd4a3ea28e6-005fc8f526
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-md5;host;x-amz-acl;x-amz-content-sha256;x-amz-date,
        Signature=8b3ce119c83756ee359daadac6db32a91d85ce532949d70b86946f084cb1c433
      Content-Md5:
      - 1B2M2Y8AsgTpgAmY7PhCfg==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Acl:
      - private
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20201203T142439Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?acl=
    method: PUT
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 03 Dec 2020 14:24:39 GMT
      X-Amz-Id-2:
      - tx59b419f3b7024f4cae617-005fc8f527
      X-Amz-Request-Id:
      - tx59b419f3b7024f4cae617-005fc8f527
    status: 200 OK
    code: 200
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6b0e806f3726fc59e261e4ab0c4ec630378ff18590ffda3d3c5bccc3bc46c7a1
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142439Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txa065e4ed979147f5a2768-005fc8f527</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:39 GMT
      X-Amz-Id-2:
      - txa065e4ed979147f5a2768-005fc8f527
      X-Amz-Request-Id:
      - txa065e4ed979147f5a2768-005fc8f527
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6b0e806f3726fc59e261e4ab0c4ec630378ff18590ffda3d3c5bccc3bc46c7a1
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142439Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx9070a7d4403b4b8cb8f70-005fc8f529</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:41 GMT
      X-Amz-Id-2:
      - tx9070a7d4403b4b8cb8f70-005fc8f529
      X-Amz-Request-Id:
      - tx9070a7d4403b4b8cb8f70-005fc8f529
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6b0e806f3726fc59e261e4ab0c4ec630378ff18590ffda3d3c5bccc3bc46c7a1
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142439Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx837b149926404bf589610-005fc8f52d</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:45 GMT
      X-Amz-Id-2:
      - tx837b149926404bf589610-005fc8f52d
      X-Amz-Request-Id:
      - tx837b149926404bf589610-005fc8f52d
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=6b0e806f3726fc59e261e4ab0c4ec630378ff18590ffda3d3c5bccc3bc46c7a1
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142439Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx21069913319041618a6bb-005fc8f536</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:54 GMT
      X-Amz-Id-2:
      - tx21069913319041618a6bb-005fc8f536
      X-Amz-Request-Id:
      - tx21069913319041618a6bb-005fc8f536
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=1c78cc6bcc607371ecba89863a9ca8a12742561ae379af675f8c50d32ae0f348
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142454Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx21f42b067ed649eaba1cd-005fc8f536</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:54 GMT
      X-Amz-Id-2:
      - tx21f42b067ed649eaba1cd-005fc8f536
      X-Amz-Request-Id:
      - tx21f42b067ed649eaba1cd-005fc8f536
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=1c78cc6bcc607371ecba89863a9ca8a12742561ae379af675f8c50d32ae0f348
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142454Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txdf0171d71f4f41289e057-005fc8f538</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:24:56 GMT
      X-Amz-Id-2:
      - txdf0171d71f4f41289e057-005fc8f538
      X-Amz-Request-Id:
      - txdf0171d71f4f41289e057-005fc8f538
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=1c78cc6bcc607371ecba89863a9ca8a12742561ae379af675f8c50d32ae0f348
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142454Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txa66725d63495420788d90-005fc8f53c</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:00 GMT
      X-Amz-Id-2:
      - txa66725d63495420788d90-005fc8f53c
      X-Amz-Request-Id:
      - txa66725d63495420788d90-005fc8f53c
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=1c78cc6bcc607371ecba89863a9ca8a12742561ae379af675f8c50d32ae0f348
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142454Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txa16e9638daa944fe9b64e-005fc8f544</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:08 GMT
      X-Amz-Id-2:
      - txa16e9638daa944fe9b64e-005fc8f544
      X-Amz-Request-Id:
      - txa16e9638daa944fe9b64e-005fc8f544
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=349f832fde84d6e49d98dd23a9dc4afde1a53cbc9457e6592f3574df90d01c50
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142508Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txf0d50e3c69d14f9488560-005fc8f544</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:08 GMT
      X-Amz-Id-2:
      - txf0d50e3c69d14f9488560-005fc8f544
      X-Amz-Request-Id:
      - txf0d50e3c69d14f9488560-005fc8f544
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=349f832fde84d6e49d98dd23a9dc4afde1a53cbc9457e6592f3574df90d01c50
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142508Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txe48064276ac843689eb9c-005fc8f546</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:10 GMT
      X-Amz-Id-2:
      - txe48064276ac843689eb9c-005fc8f546
      X-Amz-Request-Id:
      - txe48064276ac843689eb9c-005fc8f546
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=349f832fde84d6e49d98dd23a9dc4afde1a53cbc9457e6592f3574df90d01c50
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142508Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx10a0dacbf1464efc912f6-005fc8f54a</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:14 GMT
      X-Amz-Id-2:
      - tx10a0dacbf1464efc912f6-005fc8f54a
      X-Amz-Request-Id:
      - tx10a0dacbf1464efc912f6-005fc8f54a
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=349f832fde84d6e49d98dd23a9dc4afde1a53cbc9457e6592f3574df90d01c50
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142508Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txc5e5dda2601244c7a36d1-005fc8f552</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:23 GMT
      X-Amz-Id-2:
      - txc5e5dda2601244c7a36d1-005fc8f552
      X-Amz-Request-Id:
      - txc5e5dda2601244c7a36d1-005fc8f552
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=4e66b9b21f44cd213763318200d6eb7fe8232e19826e8500b6ad30595c5db3ef
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142523Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txcb235cf7858043808eb5c-005fc8f553</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:23 GMT
      X-Amz-Id-2:
      - txcb235cf7858043808eb5c-005fc8f553
      X-Amz-Request-Id:
      - txcb235cf7858043808eb5c-005fc8f553
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=4e66b9b21f44cd213763318200d6eb7fe8232e19826e8500b6ad30595c5db3ef
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142523Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>txc6aa3fdf0b7c443087e3e-005fc8f555</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:25 GMT
      X-Amz-Id-2:
      - txc6aa3fdf0b7c443087e3e-005fc8f555
      X-Amz-Request-Id:
      - txc6aa3fdf0b7c443087e3e-005fc8f555
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=4e66b9b21f44cd213763318200d6eb7fe8232e19826e8500b6ad30595c5db3ef
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142523Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx751c0f0be350427db2da4-005fc8f559</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:29 GMT
      X-Amz-Id-2:
      - tx751c0f0be350427db2da4-005fc8f559
      X-Amz-Request-Id:
      - tx751c0f0be350427db2da4-005fc8f559
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><CORSRule><AllowedMethod>PUT</AllowedMethod><AllowedMethod>POST</AllowedMethod><AllowedOrigin></AllowedOrigin><ExposeHeader>x-amz-server-side-encryption</ExposeHeader><ExposeHeader>ETag</ExposeHeader><MaxAgeSeconds>3000</MaxAgeSeconds><AllowedHeader>*</AllowedHeader></CORSRule></CORSConfiguration>
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=content-length;content-md5;host;x-amz-content-sha256;x-amz-date,
        Signature=4e66b9b21f44cd213763318200d6eb7fe8232e19826e8500b6ad30595c5db3ef
      Content-Length:
      - "365"
      Content-Md5:
      - VfPkaWFowrvJ8tkwt9YPfw==
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - 312bcd4b993bead7d20d3aa4c8767025620c21df8a0bd6f5dfbf55279797b8d2
      X-Amz-Date:
      - 20201203T142523Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/?cors=
    method: PUT
  response:
    body: |-
      <?xml version='1.0' encoding='UTF-8'?>
      <Error><Code>InternalError</Code><Message>We encountered an internal error. Please try again.</Message><RequestId>tx2b65a9c0d45f4574a4891-005fc8f561</RequestId><Reason>'NoneType' object has no attribute 'count'</Reason></Error>
    headers:
      Content-Type:
      - application/xml
      Date:
      - Thu, 03 Dec 2020 14:25:37 GMT
      X-Amz-Id-2:
      - tx2b65a9c0d45f4574a4891-005fc8f561
      X-Amz-Request-Id:
      - tx2b65a9c0d45f4574a4891-005fc8f561
    status: 500 Internal Server Error
    code: 500
    duration: ""
- request:
    body: ""
    form: {}
    headers:
      Authorization:
      - AWS4-HMAC-SHA256 Credential=SCW8XT5JRAV4B0WQSHPE/20201203/fr-par/s3/aws4_request,
        SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=da8dccf6b1251f912f98b940a9bdcf3edf7d40ab91f8186782c8dcaaa3b307ca
      User-Agent:
      - aws-sdk-go/1.34.32 (go1.15.5; darwin; amd64)
      X-Amz-Content-Sha256:
      - e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      X-Amz-Date:
      - 20201203T142538Z
    url: https://test-acc-scaleway-object-bucket-cors-empty-origin.s3.fr-par.scw.cloud/
    method: DELETE
  response:
    body: ""
    headers:
      Content-Length:
      - "0"
      Content-Type:
      - text/html; charset=UTF-8
      Date:
      - Thu, 03 Dec 2020 14:25:38 GMT
      X-Amz-Id-2:
      - tx0e92ff92e07847c1a0905-005fc8f562
      X-Amz-Request-Id:
      - tx0e92ff92e07847c1a0905-005fc8f562
    status: 204 No Content
    code: 204
    duration: ""
